pages/spaces/space.js

```diff
@@ -1,56 +1,12 @@
 import React from 'react'
```

Comment:
please don't make push these changes into a pr tho. Pull requests are meant to be merged with the main branch. This and the deleted files above will break the main branch.